xdsl/dialects/riscv.py

```diff
@@ -139,7 +143,16 @@ def abi_name(self):
         return self.data.name
 
 
-class RdRsRsOperation(IRDLOperation, ABC):
+@irdl_attr_definition
+class LabelAttr(Data[str]):
+    name = "riscv.label"
+
+
+class _RISCVOp(Operation, ABC):
```

Comment:
I think this should be ok to make public
```suggestion
class RISCVOp(Operation, ABC):
```